90zfsbootmenu/hook/zfsbootmenu-preinit.sh

```diff
@@ -17,12 +17,11 @@ if ! [[ ${control_term} =~ ${tty_re} ]]; then
 fi
 
 export BASE="/zfsbootmenu"
-mkdir -p "${BASE}"
 
 # shellcheck disable=SC2154
 cat >> "/etc/zfsbootmenu.conf" <<EOF
 # BEGIN additions by zfsbootmenu-preinit.sh
-export BASE="/zfsbootmenu"
+export BASE="${BASE}"
```

Comment:
For that final *chefs kiss* of protection, should this be `export BASE="${BASE:=/zfsbootmenu}"` as well?

Reply:
You're killing me, Smalls